.github/workflows/cypress.yml

```diff
@@ -33,5 +33,8 @@ jobs:
     - name: Set up WP environment
       run: npm run wp-env start
 
+    - name: Create pages
+      run: npx wp-env run tests-cli "wp post create --post_type=page --post_title='Accessible page' --post_status='publish'" && npx wp-env run tests-cli "wp post create --post_type=page --post_title='Page to redirect' --post_status='publish'"
```

Comment:
This command is only executed inside GitHub Actions. Still required to run this manually for testing on a local env.

Maybe worth putting it as a new NPM script to package.json? Will be possible to execute the script as `precypress:open` and `precypress:run`. Will need to run `npx wp-env clean tests` before post create commands to prevent duplication.

Reply:
Ah sorry, it's my mistake. This PR isn't ready for review yet as the tests are failing on Github actions.
I planned to add the command to NPM after the tests passed, but thanks for reviewing though.